spdx-utils/src/main/kotlin/SpdxLicense.kt

```diff
@@ -491,7 +491,7 @@ enum class SpdxLicense(
         /**
          * The version of the license list.
          */
-        const val LICENSE_LIST_VERSION = "3.10"
+        const val LICENSE_LIST_VERSION = "3.10-9-ge0f7167"
```

Comment:
License list number are always just digits, current one is 3.10 see https://spdx.org/licenses/. Won't this change form 3.10 to 3.10-9-ge0f7167 cause an invalid license list number in SPDX output docs?

Reply:
> License list number are always just digits

In the SPDX documents this is true. But here we have a constant which holds the exact version name.
The SPDX reporter will still output "3.10", it is covered also by a test.
IIRC this was discussed before and agreed upon as-is.